decidim-elections/config/locales/en.yml

```diff
@@ -1291,6 +1297,7 @@ en:
           day_placeholder: DD
           document_number: Document number
           document_number_placeholder: ID number
+          document_type: Tipus de document
```

Comment:
```suggestion
          document_type: Document type
```

Uops catalan string, this needs to be changed

Reply:
I see that have some failing specs that are related to this, so they'll pass when this is changed https://github.com/decidim/decidim/runs/5705110331?check_suite_focus=true

Reply:
Ouf! Rookie mistake. Thanks!

Reply:
It seems like it's still there, can you check it again?